module/validation/seal_validator.go

```diff
@@ -212,7 +212,7 @@ func (s *sealValidator) Validate(candidate *flow.Block) (*flow.Seal, error) {
 				// because we still want to test that the above code doesn't panic.
 				// TODO: this is only here temporarily to ease the migration to new chunk
 				// based sealing.
-				if s.requiredChunkApprovals == 0 {
+				if s.requiredApprovalsForSealVerification == 0 {
 					log.Warn().Msgf("payload includes invalid seal, continuing validation (%x): %s", seal.ID(), err.Error())
 				} else {
 					return nil, fmt.Errorf("payload includes invalid seal (%x): %w", seal.ID(), err)
```

Comment:
better also include the result id in the error message
```suggestion
					return nil, fmt.Errorf("payload includes invalid seal (%x): %w", seal.ID(), err)
```